spec/intl.html

```diff
@@ -1338,6 +1338,259 @@ <h1>CalendarEraYear ( _calendar_, _dateLike_ )</h1>
+            CalendarDateDaysInWeek (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of days in the given week in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinmonth" type="abstract operation">
+          <h1>
+            CalendarDateDaysInMonth (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of days in the given month in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatedaysinyear" type="abstract operation">
+          <h1>
+            CalendarDateDaysInYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatemonthsinyear" type="abstract operation">
+          <h1>
+            CalendarDateMonthsInYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns the number of months in the given year in the calendar represented by _calendar_.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardateinleapyear" type="abstract operation">
+          <h1>
+            CalendarDateInLeapYear (
+              _calendar_: a String,
+              _date_: a %Temporal.PlainDate%,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a date-like object _date_ and returns *true* if it falls within a leap year for the calendar represented by _calendar_ and *false* otherwise.</dd>
+          </dl>
+        </emu-clause>
+
+        <emu-clause id="sec-temporal-calendardatefields" type="abstract operation">
+          <h1>
+            CalendarDateFields (
+              _calendar_: a String,
+              _fields_: a List of Strings,
+            )
+          </h1>
+          <dl class="header">
+            <dt>description</dt>
+            <dd>It takes a list of standard fields in _fields_ and returns a new list by adding relevant calendar-specific fields for the calendar represented by _calendar_.</dd>
```

Comment:
(Copying from my earlier comment that got lost.) There should probably be more commentary about what this operation does and what implementations are supposed to do with it. See https://github.com/tc39/proposal-temporal/issues/1899#issuecomment-962859406 for more info.

Same for mergeFields below.